flashinfer/gdn_decode.py

```
@@ -1493,9 +1600,10 @@ def run_gdn_decode_kernel_small_batch_nontranspose(
     use_qk_l2norm: cutlass.Constexpr[bool],
     stream: cuda.CUstream,
 ):
-    pool_size, hv_dim, k_dim, v_dim = h0_source.layout.shape
-    n_indices = h0_indices.layout.shape[0]
-    batch_size = n_indices * hv_dim
+    # h0_source is flattened to [B*HV, K, V] to ensure proper alignment for SIMT async copy
+    batch_hv_dim, k_dim, v_dim = h0_source.layout.shape
```

Ruff (0.14.14)

Unpacked variable `k_dim` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)